releases/release-1.25/README.md

```diff
@@ -57,34 +57,35 @@ The 1.25 release cycle is as follows:
 | Schedule finalized                                            | Lead | Friday 27th May 2022                                                                                            | week 1   | |
 | Team finalized                                                | Lead | Friday 27th May 2022                                                                                            | week 1   | |
 | Start Release Notes Draft                                     | Release Notes Lead | Tuesday 7th June 2022                                                                                           | week 3   | |
+| 1.25.0-alpha.1 released                                       | Branch Manager | Wednesday 15th June 2022                                                                                             | Week 4   | |
+
```

Comment:
```suggestion
```

This breaks the table, so we have to remove this line :blush:

Reply:
thank you! I obviously had a very messy markdown 😅